Resources/Контробандист/Прото рольки.yml

```
@@ -0,0 +1,47 @@
+
+- type: startingGear
+  id: SmugglerGear
+  equipment:
+    ears: ClothingHeadsetAltSyndicate
+    id: SyndiPDA
+    belt: ClothingBeltSyndieHolster
+    pocket1: RubberStampSyndicate
+    mask: ClothingMaskNeckGaiter
+  storage:
+    back:
+    - BoxSurvivalNFOxygenExtended
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)